lib/gen_stage.ex

```diff
@@ -2407,7 +2407,7 @@ defmodule GenStage do
 
   ## Consumer helpers
 
-  defp consumer_init_subscribe(producers, stage, continue) do
+  defp consumer_init_subscribe(producers, stage, continue_or_hibernate) do
     fold_fun = fn
       to, {:ok, stage, :no_continue} ->
```

Comment:
We don't need this clause if we have a case at the end.

Reply:
You are right, I'll remove it.